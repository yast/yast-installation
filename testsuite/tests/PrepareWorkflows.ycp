{

// testedfiles: AddOnProduct.ycp

include "testsuite.ycp";

map READ = $[
    "target" : $[
	"size" : 0,
    ],
];

map WRITE = $[];

map EXEC = $[
    "target" : $[
	"bash_output" : $["exit" : 0, "stdout" : "", "stderr" : ""],
    ],
];

TESTSUITE_INIT([READ, WRITE, EXEC], nil);

import "AddOnProduct";

list<map> workflows = [
  $[
    "defaults":$["archs":"i386,x86_64"],
    "label":"Preparation",
    "mode":"installation,update",
    "modules":[
      $["arguments":$["first_run":"yes"], "enable_back":"no", "enable_next":"yes", "label":"Language", "name":"language", "retranslate":true],
      $["label":"Perform Installation", "name":"finish"]
    ],
    "stage":"initial"
  ]
];

TEST(``(AddOnProduct::PrepareWorkflows(workflows)), [], nil);

}
