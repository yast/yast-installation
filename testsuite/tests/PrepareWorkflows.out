Return	[$["defaults":$["archs":"i386"], "label":"Preparation", "mode":"installation", "modules":[$["arguments":$["first_run":"yes"], "enable_back":"no", "enable_next":"yes", "label":"Language", "name":"language", "retranslate":true], $["label":"Perform Installation", "name":"finish"]], "stage":"initial"], $["defaults":$["archs":"x86_64"], "label":"Preparation", "mode":"installation", "modules":[$["arguments":$["first_run":"yes"], "enable_back":"no", "enable_next":"yes", "label":"Language", "name":"language", "retranslate":true], $["label":"Perform Installation", "name":"finish"]], "stage":"initial"], $["defaults":$["archs":"i386"], "label":"Preparation", "mode":"update", "modules":[$["arguments":$["first_run":"yes"], "enable_back":"no", "enable_next":"yes", "label":"Language", "name":"language", "retranslate":true], $["label":"Perform Installation", "name":"finish"]], "stage":"initial"], $["defaults":$["archs":"x86_64"], "label":"Preparation", "mode":"update", "modules":[$["arguments":$["first_run":"yes"], "enable_back":"no", "enable_next":"yes", "label":"Language", "name":"language", "retranslate":true], $["label":"Perform Installation", "name":"finish"]], "stage":"initial"]]
